.evergreen-functions.yml

```diff
@@ -542,50 +542,9 @@ functions:
         working_dir: src/github.com/mongodb/mongodb-kubernetes
         binary: scripts/release/pipeline.sh
         env:
-          IMAGE_NAME: ${IMAGE_NAME}
-          FLAGS: ${FLAGS}
-
-  pipeline_agent:
-    - command: subprocess.exec
-      retry_on_failure: true
-      type: setup
-      params:
-        shell: bash
-        <<: *e2e_include_expansions_in_env
-        working_dir: src/github.com/mongodb/mongodb-kubernetes
-        binary: scripts/release/pipeline_agent.sh
-        env:
-          FLAGS: ${FLAGS}
-
-  pipeline_ops_manager:
-    - command: subprocess.exec
-      retry_on_failure: true
-      type: setup
-      params:
-        shell: bash
-        <<: *e2e_include_expansions_in_env
-        working_dir: src/github.com/mongodb/mongodb-kubernetes
-        binary: scripts/release/pipeline_ops_manager.sh
-
-  pipeline_readiness_probe:
-    - command: subprocess.exec
-      retry_on_failure: true
-      type: setup
-      params:
-        shell: bash
-        <<: *e2e_include_expansions_in_env
-        working_dir: src/github.com/mongodb/mongodb-kubernetes
-        binary: scripts/release/pipeline_readiness_probe.sh
-
-  pipeline_version_upgrade_hook:
-    - command: subprocess.exec
-      retry_on_failure: true
-      type: setup
-      params:
-        shell: bash
-        <<: *e2e_include_expansions_in_env
-        working_dir: src/github.com/mongodb/mongodb-kubernetes
-        binary: scripts/release/pipeline_version_upgrade_hook.sh
+          IMAGE_NAME: ${image_name}
+          BUILD_SCENARIO_OVERRIDE: ${build_scenario}
```

Comment:
q: did we add `BUILD_SCENARIO_OVERRIDE` so that we can build the images for different build scenarios manually?